src/components/navbar/ReaderNavBar.tsx

```diff
@@ -46,10 +46,13 @@ const useStyles = (settings: IReaderSettings) => makeStyles({
         minWidth: '300px',
         height: '100vh',
         overflowY: 'auto',
-        backgroundColor: '#0a0b0b',
+        backgroundColor: theme.palette.mode === 'dark' ? '#0a0b0b' : '#fff',
 
         '& header': {
-            backgroundColor: '#363b3d',
+            backgroundColor:
+            // the shade of white is from the main navbar color
+            // it would be better if we could access this color from the mui theme
+            theme.palette.mode === 'dark' ? '#363b3d' : '#f5f5f5',
```

Comment:
You can, from the theme object itself: https://mui.com/customization/default-theme/

Reply:
I can't seem to find the exact color as the one used in the Navbar, I'll just use one of the Greys for now